From 3d3611791555b6f002aa9836ee89a3a14ca0b163 Mon Sep 17 00:00:00 2001
From: Mattias Nissler <mnissler@google.com>
Date: Fri, 21 Apr 2017 23:34:26 +0000
Subject: [PATCH 0112/1240] VENDOR: broadcom: bcmdhd: Use sdio_reset_comm from
 common kernel

A proper definition of sdio_reset_comm() was brought in when merging
Android-specific common kernel changes into Brillo's common kernel. We
can now the hack compile fix hack in bcmsdh_sdmmc.c.

Bug: 25923642
Patchset: wlan0 up on kylin board, iw is normal connect the AP.

Signed-off-by: Mattias Nissler <mnissler@google.com>
Change-Id: I7fea434b7f173b25acab2942eba106c2da542457
---
 drivers/net/wireless/bcmdhd/bcmsdh_sdmmc.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/net/wireless/bcmdhd/bcmsdh_sdmmc.c b/drivers/net/wireless/bcmdhd/bcmsdh_sdmmc.c
index 04a3e0d..199522e 100644
--- a/drivers/net/wireless/bcmdhd/bcmsdh_sdmmc.c
+++ b/drivers/net/wireless/bcmdhd/bcmsdh_sdmmc.c
@@ -62,14 +62,7 @@ static void IRQHandler(struct sdio_func *func);
 static void IRQHandlerF2(struct sdio_func *func);
 #endif /* !defined(OOB_INTR_ONLY) */
 static int sdioh_sdmmc_get_cisaddr(sdioh_info_t *sd, uint32 regaddr);
-#if 0 
 extern int sdio_reset_comm(struct mmc_card *card);
-#else
-int sdio_reset_comm(struct mmc_card *card)
-{
-	return 0;
-}
-#endif
 
 #define DEFAULT_SDIO_F2_BLKSIZE		512
 #ifndef CUSTOM_SDIO_F2_BLKSIZE
-- 
2.7.4

